### PR TITLE
Styles for the heading elements in blog posts.

### LINKED ISSUE #65: Bug: Headers are not rendering
### Describe the bug

H1, H2, etc ate not rendering on the page. 

Screen Shot 2022-10-07 at 7 21 40 PM

### Steps to reproduce

Screen Shot 2022-10-07 at 7 22 14 PM

### Affected services

opensauced.pizza

### Platforms

_No response_

### Browsers

_No response_

### Environment

_No response_

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by shamimbinnur**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
Added styles for the Heading elements(H1, H2, H3, etc) in blog posts in order to make them stand out in the blog content.

## Related Tickets & Documents
fixes #65 

## Mobile & Desktop Screenshots/Recordings
Screenshot 2022-10-24 at 8 25 32 AM

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Review by bdougie — APPROVED** [review]

File: styles/globals.css [hunk 1]
```diff
@@ -34,9 +34,27 @@ a {
 }
 
 @layer components {
-  .styledDocument a{
+  .styledDocument a {
     @apply font-semibold text-darkBlue;
   }
+  .styledDocument h1 {
+    @apply text-[2em] font-semibold py-6 leading-10
```
Comment (bdougie) [inline styles/globals.css:41]: Why em and not tailwind here?

Comment (shamimbinnur) [reply styles/globals.css:41]: @bdougie The tailwind class doesn't cover all the values I have used here. These em values are exactly the same value that h1, h2, etc inherit by default from HTML. So I considered em here in order to maintain consistency.

Comment (bdougie) [reply styles/globals.css:41]: TBH, we don't need anything past h3. This is setup for scalability but doesn't take into consideration that this page has not been designed by @Saucy-Christian.

I'd prefer if we used the defaults from tailwind than begging building our own custom framework.

Comment (shamimbinnur) [reply styles/globals.css]: @bdougie Replaced with tailwind classes now.

**Review by bdougie — APPROVED** [review]

**Pull request merged**